plugins/web/opentelemetry-instrumentation-user-interaction/src/instrumentation.ts

```diff
@@ -36,6 +36,22 @@ import { VERSION } from './version';
 const ZONE_CONTEXT_KEY = 'OT_ZONE_CONTEXT';
 const EVENT_NAVIGATION_NAME = 'Navigation:';
 
+/**
+ * Most browser provide event listener api via EventTarget in prototype chain.
+ * Exception to this is IE 11 which has it on the prototypes closest to EventTarget:
+ *
+ * * - has addEventListener in IE
+ * ** - has addEventListener in all other browsers
+ * ! - missing in IE
+ *
+ * HTMLElement -> Element -> Node * -> EventTarget **! -> Object
+ * Document -> Node * -> EventTarget **! -> Object
+ * Window * -> WindowProperties ! -> EventTarget **! -> Object
+ */
+const EVENT_TARGETS = window.EventTarget
+  ? [EventTarget.prototype]
+  : [Node.prototype, Window.prototype];
```

Comment:
pls correct me if I'm wrong but wouldn't this produce a double events on IE ?

Reply:
It wouldn't, much like pre-#537 targeted addEventListener calls to objects that had HTMLElement in it's prototype chain (HTMLAnchorElement -> **HTMLElement** -> Element -> Node -> EventTarget (if not IE) -> Object), anything in a different prototype chain that didn't have patched HTMLElement's addEventListener would not be affected (so document.addEventListener didn't produce spans as it's chain is Document -> Node -> EventTarget -> Object).

This behavior can actually be confirmed in any browser: https://jsfiddle.net/t2t2/cpw8svj4/

Same code ran in IE: (ofc jsfiddle is broken in IE)

<img width="517" alt="image" src="https://user-images.githubusercontent.com/1122555/130479959-630348ab-027c-4d4a-bfa5-ca2534c182f2.png">

(So in summary in IE Node.prototype has a reference to native addEventListener and Window.prototype has a reference to native addEventListener, changing one reference doesn't affect the other)